contracts/peg/VanillaPSM.sol

```diff
@@ -0,0 +1,12 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+pragma solidity ^0.8.4;
+
+import {BasePSM} from "./BasePSM.sol";
+import {IPCVDeposit} from "../pcv/IPCVDeposit.sol";
+import "@openzeppelin/contracts/token/ERC20/utils/SafeERC20.sol";
```

Comment:
let's do an explicit import here and specify what contract/interface we want imported